.actrc

```diff
@@ -0,0 +1,7 @@
+--action-offline-mode
+--container-architecture linux/amd64
+--platform macos-14=-self-hosted
+--pull=false
+--remote-name phatblat
```

Comment:
This is necessary to avoid "remote not found errors" when the remote name is not `origin`.

> [pr-checks/lint] unable to get git repo (githubInstance: github.com; remoteName: origin, repoPath: /Users/phatblat/dev/just/just-experiments): remote not found

I like to use the owner name for the remote for OSS repos. `act` needs help finding the remote with the `--remote-name` option

```
❯ git remote -v
phatblat	git@github.com:phatblat/just-experiments.git (fetch)
phatblat	git@github.com:phatblat/just-experiments.git (push)
```